addons/analytic/static/src/components/analytic_distribution/analytic_distribution.js

```diff
@@ -329,41 +329,30 @@ export class AnalyticDistribution extends Component {
         return this.state.showDropdown && !!this.dropdownRef.el;
     }
 
-    applicabilityStatus(group_id) {
+    titleCase(word) {
+        return `${word.charAt(0).toUpperCase()}${word.slice(1).toLowerCase()}`;
+    }
+
+    statusDescription(group_id) {
         const group = this.list[group_id];
-        const status = this.groupStatus(group_id);
-        let description;
-        switch(status){
-            case "gray":
-                description = this.env._t("Editing (OK)");
-                break;
-            case "orange": {
-                description = this.env._t("Editing (Incomplete)");
-                break;
-            }
-            case "red": {
-                description = this.env._t("Invalid");
-                break;
-            }
-            case "green": {
-                description = this.env._t("OK");
-                break;
-            }
-        }
-        return `${group.applicability.charAt(0).toUpperCase()}${group.applicability.slice(1)} - ${description}`;
+        const applicability = this.env._t(`${this.titleCase(group.applicability)}`);
+        const description = this.env._t(`${this.titleCase(this.groupStatus(group_id))}`);
```

Comment:
You can't translate dynamic strings

Reply:
solved with some ugly `switch/case`'s - or we can remove the translations?